lib/model/team.rb

```diff
@@ -6,12 +6,13 @@ class Team
               :stadium,
               :link
 
-  def initialize(data)
-    @team_id = data[:team_id].to_i
-    @franchise_id = data[:franchiseId].to_i
+  def initialize(data, parent)
+    @team_id = data[:team_id]
```

Comment:
In the other 2 classes we made these integers so the tests didn't have to be strings. Do we want to do the same here?